corehq/apps/app_manager/models.py

```diff
@@ -5675,13 +5674,13 @@ def get_latest_master_releases_versions(self):
         if self.domain_link:
             versions = get_latest_master_releases_versions(self.domain_link)
             # Use self.get_master_app_briefs to limit return value by family_id
-            master_ids = [b.id for b in self.get_master_app_briefs()]
-            return {key: value for key, value in versions.items() if key in master_ids}
+            origin_ids = [b.id for b in self.get_master_app_briefs()]
```

Comment:
Is this right? I think here this is actually `master_id` since it's from a linked app, right?

Reply:
I see what you mean. They're origin ids, but they're the origin ids of upstream apps, and the upstream part is more relevant. Updated.

Reply:
🌊